compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -355,6 +355,7 @@ class ConvertNCHWToNHWC final : public luci::CircleNodeMutableVisitor<bool>
     node->shape_status(luci::ShapeStatus::VALID);
 
     loco::shape_erase(node);
+    node->shape_status(luci::ShapeStatus::UNDEFINED);
```

Comment:
if `loco::shape_erase(node);` is to be removed, could you add a `TODO: remove loco::shape_erase()` ?